test/e2e/install/install_test.go

```diff
@@ -137,7 +124,11 @@ func getTemplateYamlDir() string {
 }
 
 func generateYamls(imageTag string, registryAddress string) {
-	bashcmd := fmt.Sprintf("../../../scripts/generate-deploy-yamls.sh -o ../../../test-yamls -t %s -r %s test", imageTag, registryAddress)
+	bashcmd := fmt.Sprintf("../../../scripts/generate-deploy-yamls.sh -o ../../../test-yamls -t %s", imageTag)
+	if registryAddress != "" {
+		bashcmd = bashcmd + " -r " + registryAddress
+	}
```

Comment:
The trouble is it that form doesn't work if registryAddress is a zero-length string, because -r will get "test". 
Also "test" has to end the command because of the way the script parses arguments.
I could raise a Jira to suggest changing the argument handling in generate-deploy-yamls.sh

Reply:
not if you put the apostrophes in

Reply:
Ah .. I see